packages/generator/src/utils/get-latest-version.ts

```diff
@@ -17,7 +17,8 @@ export const getLatestVersion = async (
   dependency: string,
   templateVersion: string = '',
 ): Promise<Fallbackable<string>> => {
-  const major = templateVersion.replace('.x', '')
+  const toBeReplaced = templateVersion.substring(templateVersion.indexOf('.'))
```

Comment:
could this be renamed to something like `minorPatch`, which says more about the value and fits more to the next line, where the major version is extracted from the whole version string: `const major = templateVersion.replace(minorPatch, '')`?